backend/src/main/java/com/tradomator/model/Balance.java

```diff
@@ -0,0 +1,18 @@
+package com.tradomator.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class Balance {
```

Comment:
Thi class can probably be replaced by a record (since Java 14) and would look like this:
```java
public record Balance(String asset, String free, String locked) {
}
```

Reply:
really nice approach to reach cleaner code. I tried to change . It returns for asset instanced field is not allowed in records. Change into static still would cause issues. Will keep now and remember next time building entities. Thanks for showing me this new option :-)